exporters/elasticsearch/include/opentelemetry/exporters/elasticsearch/es_log_exporter.h

```diff
@@ -117,7 +117,7 @@ class ElasticsearchLogExporter final : public sdklogs::LogExporter
   ElasticsearchExporterOptions options_;
 
   // Object that stores the HTTP sessions that have been created
-  std::unique_ptr<ext::http::client::SessionManager> session_manager_;
+  std::unique_ptr<ext::http::client::HttpClient> session_manager_;
```

Comment:
Rename the variable name to `http_client_` too?

Reply:
yeah, this needs to be done in both cc and h. I was thinking to change it once @maxgolov is fine with the current changes ( to avoid reverting back).  But thanks for noticing it. I will add this as pending change in PR description.